packages/cli/src/lib/templates/storybook.preview.tsx.template

```diff
@@ -1,18 +1,19 @@
 import * as React from 'react'
 
 import type { GlobalTypes } from '@storybook/csf'
-import type { StoryFn, StoryContext } from '@storybook/react'
+import type { Preview, StoryFn, StoryContext } from '@storybook/react'
 
-/** @type { import("@storybook/csf").GlobalTypes } */
+/** @see {@link https://storybook.js.org/docs/7/essentials/toolbars-and-globals#global-types-and-the-toolbar-annotation | Global types and the toolbar annotation}  */
 export const globalTypes: GlobalTypes = {}
 
 /**
  * An example, no-op storybook decorator. Use a function like this to create decorators.
- * @param { import("@storybook/react").StoryFn} StoryFn
- * @param { import("@storybook/react").StoryContext} context
-*/
+ * @see {@link https://storybook.js.org/docs/7/essentials/toolbars-and-globals#create-a-decorator | Create a decorator} 
+ */
 const _exampleDecorator = (StoryFn: StoryFn, _context: StoryContext) => {
   return <StoryFn />
 }
 
-export const decorators = []
```

Comment:
Did you mean to remove this?

[https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#removed-global-client-apis](This section) of Storybook's migration guide says to use `export const decorators = []` in the preview file.

But, I haven't used Storybook much, and never used SB 7. So I could be totally wrong here

Reply:
The link you are referring to is a subsection of ["7.0 Addon authors changes"](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#70-addon-authors-changes) :wink: 

As the link i added in the commit message states:
> Storybook 7.0 supports a default export in .storybook/preview.js that should contain all of its annotations. The previous format is still compatible, but **the default export will be the recommended way going forward.**

Using this kind of config for a while now, so i can confirm that it works (and also is like auto-pilot b/c we now have the `Preview` type auto-completion).

Reply:
I thought we were supposed to have *both* the `decorators` export and the default `preview` export. But that was more a guess than anything.

Reply:
I found this: https://storybook.js.org/docs/writing-stories/decorators#global-decorators

So maybe, to keep it more similar to what we had before, we should do
```
const preview: Preview = {
  decorators: []
}
```

Reply:
Fine with me.